### DISCUSSION TITLE
WER validation does not drop down.

### DISCUSSION THREAD

**Discussion by amitabduls15** [NVIDIA-NeMo/NeMo#2981] [Q&A]
hi I have a question regarding the ASR model I am currently training on.
I'm trying to make an ASR model from the Indonesian common voice from Mozilla

the following is the config I use:
[exp1.txt](https://github.com/NVIDIA/NeMo/files/7319416/exp1.txt)

![image](https://user-images.githubusercontent.com/38752309/136730793-dbe215ed-3cf5-4beb-98e3-259e50af8d3e.png)

up to epoch 300 the value of WER on the validation data is only in the range of 1, is this because the config I use is wrong?

I try to see the sample rate data common voice has 20khz but in the config I use the config models train_ds, val_ds and tes_ds 16khz. on preprocessing using 16khz. does this have any effect?

there any suggestion i can do to improve my model? or do I have to preprocess the common voice data to all 16khz? or I change the base model ASR that I'm using now?

thank you

**Comment by titu1994** [comment]
Does the Indonesian alphabet use space tokens to denote seperation of words? If not, you should use CER instead of WER by passing model.use_cer = True.

**Reply by titu1994** [reply]
Ok if space is used then wer is the correct metric. There must be some issue with the model training recipe or the dataset which is causing the WER to plateau around 100%
